src/tests/tablet/schedule/AUserCannotUpdateIfDoesnotClickUpdate.java

```diff
@@ -0,0 +1,84 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+import static framework.common.AppConfigConstants.EXCEL_PATH;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+import framework.utils.readers.JsonReader;
+
+/**
+ * TC45: Verify if the user doesn't click update button the information is not updated
+ * @author Asael Calizaya
+ *
+ */
+public class AUserCannotUpdateIfDoesnotClickUpdate {
+	String filePath = "\\src\\tests\\meeting01.json";
```

Comment:
You should keep this or this:
String path = System.getProperty("user.dir") + EXCEL_PATH + "\\meeting01.json";